lib/term.rb

```diff
@@ -0,0 +1,17 @@
+require 'nokogiri'
+require_relative 'table'
+
+class Term < Page
```

Comment:
As some of these classes are Pages, and some are just parts, it might be clearer to be more explicit with this one and call it TermPage